.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "editor.tabSize": 2,
```

Comment:
>2 [](start = 20, length = 1)

(nit) isn't that too short?

Reply:
I prefer 4 spaces, but Vladimir chose 2 space in the clang-format file he just checked in, so that's what I went with. Maybe you can convince him to change it? :)